setup.py

```diff
@@ -22,7 +22,7 @@
 # prerequisite: setuptools
 # http://pypi.python.org/pypi/setuptools
 
-REQUIRES = ["urllib3 >= 1.15", "six >= 1.8.0", "certifi >= 14.05.14", "python-dateutil >= 2.5.3", "setuptools >= 21.0.0", "PyJWT>=1.7.1,<2", "cryptography>=2.5", "nose>=1.3.7"]
+REQUIRES = ["urllib3 >= 1.15", "six >= 1.8.0", "certifi >= 14.05.14", "python-dateutil >= 2.5.3", "setuptools >= 21.0.0", "PyJWT==2.3.0", "cryptography>=2.5", "nose>=1.3.7"]
```

Comment:
This will require users to be on _exactly_ version 2.3.0. I think a better option here is to specify between 2-3.
```suggestion
REQUIRES = ["urllib3 >= 1.15", "six >= 1.8.0", "certifi >= 14.05.14", "python-dateutil >= 2.5.3", "setuptools >= 21.0.0", "PyJWT>2,<3", "cryptography>=2.5", "nose>=1.3.7"]
```

Reply:
Yup, the change on the return type of `jwt.encode` was introduced in 2.0.0 and it's the only thing that could be considered a breaking change in the scope of this project so I guess it'd be alright to loosen the requirement a little bit.